notebook/tests/selenium/utils.py

```diff
@@ -33,6 +54,36 @@ def _wait_for(driver, locator_type, locator, timeout=10, visible=False, single=F
     return wait.until(conditional((locator_type, locator)))
 
 
+def _wait_for_multiple(driver, locator_type, locator, timeout, wait_for_n, visible=False):
+    """Waits until `wait_for_n` matching elements to be present (or visible).
+    Returns located elements when found.
+
+    Args:
+        driver: Selenium web driver instance
+        locator_type: type of locator (e.g. By.CSS_SELECTOR or By.TAG_NAME)
+        locator: name of tag, class, etc. to wait for
+        timeout: how long to wait for presence/visibility of element
+        wait_for_n: wait until this number of matching elements are present/visible
+        visible: if True, require that elements are not only present, but visible
+    """
+    wait = WebDriverWait(driver, timeout)
+
+    def multiple_found(driver):
+        try:
+            elements = driver.find_elements(locator_type, locator)
+        except WebDriverException as e:
+            raise e
```

Comment:
I see a similar pattern in some of the selenium code, but as I understand it, the try/except is redundant here - reraising the exception is the same as not catching it.

Reply:
yeah I was confused by it too, but included because the selenium conditionals did. happy to remove!